inactivity.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for moving maintainers to emeritus status when a project does not have an explicit policy of their own, and the responsibilities of the Technical Steering Committee (TSC), Hyperledger Foundation staff, and maintainers in this process.
+
+## Maintainer Inactivity
+
+This policy applies to projects that do not have an explicit maintainer inactivity policy. Where a project has an established and functioning policy, only that project's policy will apply.
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privileges is in the interest of an orderly and secure project.
+
+Activity can be code contributions, code reviews, issue reporting, or any other such activity trackable by GitHub attributed to a Hyperledger repository.
+
+When a maintainer has not had any activity in a particular project for three months they will receive a notification informing them of the inactivity policies. The means and manner of notification (email, github mentions, etc.) will be at the discretion of the TSC Chair or who the TSC Chair designates. 
+
+When a maintainer has not had any activity in a particular project for six months a proposal will be opened up to move the maintainer from active status to emeritus status. Any permissions to approve pull requests or commit code and any other such privileges associated with maintainer status will be removed.
```

Comment:
Who will open the proposal?

Reply:
Anyone from the TSC?  Or the TSC chair or who the TSC chair designates?

Reply:
I'm good with the overall proposal here and I am approving. Do we need to indicate that a majority of project maintainers should approve the request prior to merging? Or is it assumed that each project defines such policies?

(Side note: getting a majority of maintainers to chime in on a PR can be difficult, especially if there are inactive maintainers - this is one of the reasons why it helps to retire inactive maintainers. In general it is better to build consensus than have votes by majority rule, but for some contentious decisions such as retirements having a majority vote on record helps to finalize a decision).

Reply:
This is a default policy that does not involve maintainer action, simply maintainer inaction.  Hence if a maintainer ceases to be active and a PR is posted and no other maintainers approve the PR it will still merge. It doesn't need a majority of the maintainers or even a majority of those voicing an opinion.

Being a default policy if the maintainers don't like it they should update their own policy, so long as a policy exists and is functional this policy has no impact (assuming this policy passes).